cdap-ui/app/features/adapters/controllers/list-ctrl.js

```diff
@@ -32,18 +37,24 @@ angular.module(PKG.name + '.feature.adapters')
       });
 
     function pollStatus(app) {
-      dataSrc.poll({
-        _cdapNsPath: '/adapters/' + app.name + '/status'
-      }, function(res) {
-        app.status = res.status;
-      });
+      var statusParams = angular.extend({
+        app: app.name
+      }, params);
+
+      myAdapterApi.pollStatus(statusParams)
```

Comment:
What happens in the case where there are more than 1 person? The other can start the adapter too right?